modules/client-configuration/pages/supported-features-sle-micro.adoc

```diff
@@ -0,0 +1,165 @@
+[[supported-features-suse]]
+= Supported {micro} and {opensuse}{nbsp}{microos} Client Features
+
+[WARNING]
+====
+Support for {micro} and {opensuse}{nbsp}{microos} clients is provided as a technology preview for testing purposes, and not all features are fully functional at this stage.
+This feature is expected to be fully supported in a later version of {productname}.
+Do not use this feature on production systems.
+====
+
+[IMPORTANT]
+====
+The operating system you run on a client is supported by the organization that supplies the operating system.
+{micro} is supported by {suse}.
+{opensuse}{nbsp}{microos} is supported by the {suse} community.
+====
+
+
+The icons in this table indicate:
+
+* {check} the feature is available on both Salt and traditional clients
+* {cross} the feature is not available
+* {question} the feature is under consideration, and may or may not be made available at a later date
+* {trad} the feature is supported only on traditional clients
+* {salt} the feature is supported only on Salt clients.
+
+
+[cols="1,1", options="header"]
+.Supported Features on {micro} and {opensuse}{nbsp}{microos} Operating Systems
+|===
+
+| Feature
+| {micro} and {opensuse}{nbsp}{microos}
+
+| Client
+| {salt}
+
+| Operating system packages
+| {salt}
+
+| Registration
+| {salt}
+
+| Install packages
+| {salt}
+
+| Apply patches (requires CVE ID)
+| {salt}
+
+| Remote commands
+| {salt}
+
+| System package states
+| {salt}
+
+| System custom states
+| {salt}
+
+| Group custom states
+| {salt}
+
+| Organization custom states
+| {salt}
+
+| System set manager (SSM)
+| {salt}
+
+| Service pack migration
+| N/A
```

Comment:
Should we better say {question} here? Or is somebody trying this out already and we just wait for the result?

Reply:
Well, SLE Micro does not have Service Packs AFAIU, only major versions. So this should be called differently - do we want to diverge from the feature list of all other OSs?

Reply:
Well, the feature in SUMA to be used is the same. And we renamed the Tab to "Product Migration" which would even better match.
So we can think about renaming the description to Product Migration.